### PR TITLE
Closes #252 Adding Jquery speed parameters for "hide" and "show" effects

### LINKED ISSUE #252: DISCUSSION: redesign of Zen Audio Player
# Why?
As a front-end developer who's also a _little bit_ designer, I always try to think in how to make things more beautiful (for me, of course).

## The redesign
So, I've decided to make a redesign - and it's just a start idea for you guys see and tell me what do you think about it. Oh, and I don't think that the current design is bad, is just an idea for improvement. [You can check out the entire album on Imgur here](http://imgur.com/a/af4Ws).

### Home page
Home page

### Home page with search
Home page with search

### Demo page
Demo page

### Demo page with video description
Demo page with video description

### PR DESCRIPTION
**Pull request by flordeciruela**
### Important
Mark with [x] to select. Leave as [ ] to unselect.

### Motivation and Context
- [x] Why is this change required? What problem does it solve?
Improves the way a section is displayed or hidden.

- [x] If it fixes an open issue, include the text `Closes #1` (where 1 would be the issue number) to your commit message.
Closes #252

### Types of changes
What types of changes does your code introduce? Check all the boxes that apply:
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Description
- [x] I added the speed parameter "fast" in the "toggleElement" function for the Jquery effects "hide" and "show", additionally I had to deactivate the Eslint linebreak-style rule changing it to "off" because I found errors in the tests since I use Windows.

### Final checklist:
Go over all the following points and check all the boxes that apply  
If you're unsure about any of these, don't hesitate to ask. We're here to help!
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING**](.github/CONTRIBUTING.md) guidelines.
- [x] All tests passed.

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -37,10 +37,7 @@
             2,
             "all"
         ],
-        "linebreak-style": [
-            2,
-            "unix"
-        ],
+        "linebreak-style": "off",
```
Comment (shakeelmohamed) [inline .eslintrc.json]: Why this change?

Comment (flordeciruela) [reply .eslintrc.json]: because I use Windows and this caused me an error when running the tests, here I read how to correct it.

https://eslint.org/docs/rules/linebreak-style

However, I can revert to the previous configuration if they want to maintain that rule.
:)

Comment (shakeelmohamed) [reply .eslintrc.json]: I see. By default git will commit the change as an LF instead of CRLF, so it should be fine 😄
